packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "04085c80c2c84ea6b8375042d61797f7f46138fd",
-    "ref_origin" : "dcos-mesos-master-f179400b"
+    "ref": "9d1cb5ecba1a7921b067ad94092c3663adf19b24",
+    "ref_origin" : "dcos-mesos-1.2.0-rc1"
```

Comment:
I didn't realize you wanted this as an actual branch in the `/mesosphere/mesos` repo. I thought it was just a naming convention thing to refer to a sha/tag in upstream mesos. I've pushed the branch to `/mesosphere/mesos` now.